src/app/lib/config/services/ws.ts

```diff
@@ -411,32 +433,44 @@ export const service: DefaultServiceConfig = {
             id: 'burmese',
             href: 'https://www.bbc.com/burmese',
             label: 'မြန်မာသတင်းများ',
+            lang: 'my',
+            latinTransliteration: 'Myanmar satinmyar',
           },
           {
             id: 'zhongwen',
             href: 'https://www.bbc.com/zhongwen/simp',
             label: '中文新闻',
+            lang: 'zh',
```

Comment:
Does this need to match the lang of the default variant?
```suggestion
            lang: 'zh-hant',
```